tools/python/boutiques/localExec.py

```diff
@@ -122,15 +122,15 @@ def execute(self, mount_strings):
         #TODO: Test singularity runtime on cluster
         dcmd = 'singularity exec' + envString + ' -B ' + singularity_mounts + ' -W ' + launchDir + ' ' + str(conImage) + ' ./' + dsname
       else:
-        print('Unrecognized container type: \"%s\"'%conType)
+        log_data += 'Unrecognized container type: \"%s\"\n'%conType
         sys.exit(1)
       print('Executing via: ' + dcmd)
-      exit_code = self._localExecute( dcmd )
+      (stdout, stderr), exit_code = self._localExecute( dcmd )
     # Otherwise, just run command locally
     else:
-      exit_code = self._localExecute( command )
+      (stdout, stderr), exit_code = self._localExecute( command )
     # Report exit status
-    print('---/* End program output */---\nCompleted execution (exit code: ' + str(exit_code) + ')')
+    log_data += '---/* End program output */---\nCompleted execution (exit code: ' + str(exit_code) + ')\n'
```

Comment:
You don't seem to be doing anything with `log_data`, is it intentional?

Reply:
Actually, I wasn't sure what to do with it. I see value in having real-time print statements when working locally, but when running in distributed environment, I'd probably prefer having a log. So i don't know whether to keep both the print statements and the logs or just one of the two. If we do create a log, should it just be returned to the user, or should boutiques generate the file?

Reply:
Naturally, this change shouldn't have been pushed. i thought i had reverted it back to the print statements, but it seems that with all my merges, the log came back.